### PR TITLE
options: allow GODEBUG env variable

### PR DESCRIPTION
**Pull request by catenacyber**
cf https://github.com/golang/go/issues/49075

### CODE REVIEW

**Comment by catenacyber** [comment]
To be used by https://github.com/google/oss-fuzz/pull/9121

File: src/clusterfuzz/_internal/bot/fuzzers/options.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 OPTIONS_FILE_EXTENSION = '.options'
 
 # Whitelist for env variables .options files can set.
-ENV_VAR_WHITELIST = set([afl_constants.DONT_DEFER_ENV_VAR])
+ENV_VAR_WHITELIST = set([afl_constants.DONT_DEFER_ENV_VAR, "GODEBUG"])
```
Comment (oliverchang) [inline src/clusterfuzz/_internal/bot/fuzzers/options.py:31]: Please use single quotes.

**Review by oliverchang — APPROVED** [review]

File: src/clusterfuzz/_internal/bot/fuzzers/options.py (hunk 1 above)
Comment (catenacyber) [reply src/clusterfuzz/_internal/bot/fuzzers/options.py]: Ok, saw you fixed it already :-)

**Comment by catenacyber** [comment]
@oliverchang can I be sure that in the env variable `GODEBUG=cgocheck=2` was set in the log https://00f74ba44bbc7075c5d54be4d7db2ba5bed3b41377-apidata.googleusercontent.com/download/storage/v1/b/gonids-logs.clusterfuzz-external.appspot.com/o/libFuzzer_gonids_fuzz_parserule%2Flibfuzzer_asan_gonids%2F2022-12-10%2F02:22:57:051647.log?jk=AFshE3XbyM_CPUuRcxOpyHVRzQew9QtEBVoS1SbvwPGGLgAEGQpSGthFH_-MU_9cNnchk7B50nx_qY_a16AGDdbfgturhPelJHEZPgrbGiHG4btgXqYZnSOSDKlCZDErIIA3Ba8ycUwrS2QTJIYRfaxe2e4xHFMnSHLgrY2pJK4mIfJ_kL8gpzaq1i-c16iJa_6JuQ3vqNVX8GJMKPrM6mJuLI28zZzvrZp2SroJbpQpI_EFwquySJoDxQIEDQkz_ML0hRlK0gbL9_5jTOEdf83qNq4KGx56gbMdKJTsds_NwhPoAagkWLBQHp68x6MdH-YGSGOv_gOocFD4wCRj_aVuLelxtnZlALdOH2eA1u7Lz2JRc4zYEKTFndM1on4AA6JcxsZhFPx5DKRpVpYWcdMwGg_6RcILfBUTe7vQ4LaALRMMVGTFMR106MJyBSDwloItmtd5q6qeYzT1DTwnEoLC8xycw0Ir4x-JtSrzcCN9SDcjal06nmSQJC4fRKTNj6dIJXldOAqIesun9OZSDSs8FXUcNW-gasgxe5-IBH3sku7vH9C927cOKSwDU0miN_PkJPEnCNddOovqu8zj2zJbvi0JiAGt8kIaKXicr-1KJdfjyGRYmppjNfqIqewy17q2cAXvuDjwava9l2Dk7gJ9hLYOeYcTFi_08kUBPNiKPzLgm2bS-rBoDoa1oNGOqZYFHHJEZSwJYG8LEhDvArjNWhZbI0Y_MIvY-oKWqR8oBSWB3VrIixLP9sINuT3VOOm_YAUveqx3_XPShacb-wqVNAisES9McHrYsJnGoplL9qINypJwJA2JFt979SjRM8V7AgxBEpaFg-fZ_mhyzb-Abi78WlPOLO_Qnxhv1Zz87txm21XTDmJUjGjXexZ-WVPzEKGHjxZtOh4dtMyVFY5rYtlhhHzR1fgO0GWlOsaLUnDEbEkxJbP14aDYdCdTMb7-MtCZX4CMupNVlJlmmVq-X8LIY2A_fpSyrq1FjGoK_4buJKdtHfJ0O5Fuf7WYq1USqmhz0IOkau04O4LbxzNghTKjmAwwDzxk7QdjN65oHSeqMFgTzW524Mdg0jirPeMEEGt9J9XbDKAh_ABhfy0L1PJRtgfX1QIQ9ZRvrl_tlgVIFeVK6yu3ANiKRwzO8SfYpr9k_loOVTo1vZ5n8iCrOVwS0C2tsSrFuxhffJLNnw7Bpagb9i-8BYFkNzXZDZZ2WXnnr25fibtkwgyOzv6X4MwZPTGKy1fadzvcSXiTr4aDfQgBd0aavNIpDCwcb8xRZlgg7NLKq2dtc4uWSWkckhbUmq-0IQdID_haZ1o&isca=1 ?

**Pull request merged**